doc/image-handling.md

```diff
@@ -1,82 +1,63 @@
----
-discourse: 9310
----
-
 (about-images)=
 # About images
 
 ```{youtube} https://www.youtube.com/watch?v=wT7IDjo0Wgg
 ```
 
-Instances are based on images, which contain a basic operating system (for example a Linux distribution) and some other LXD-related information.
-
-## Introduction
+LXD uses an image-based workflow.
+Each instance is based on an image, which contains a basic operating system (for example, a Linux distribution) and some LXD-related information.
 
-LXD uses an image based workflow. It comes with a built-in image store
-where the user or external tools can import images.
+Images are available from remote image stores (see {ref}`remote-image-servers` for an overview), but you can also create your own images, either based on an existing instances or a rootfs image.
 
-Containers are then started from those images.
+You can copy images from remote servers to your local image store, or copy local images to remote servers.
+You can also use a local image to create a remote instance.
 
-It's possible to spawn remote instances using local images or local
-instances using remote images. In such cases, the image may be cached
-on the target LXD.
+Each image is identified by a fingerprint (SHA256).
+To make it easier to manage images, LXD allows defining one or more aliases for each image.
 
-## Sources
+## Caching
 
-LXD supports importing images from three different sources:
+When you create an instance using a remote image, LXD downloads the image and caches it locally.
+It is stored in the local image store with the cached bit set.
```

Comment:
Maybe use "indicator" or "flag" here rather than "bit", as that sounds a bit low level/technical to me (its not actually using a bit anyway ;)).